homeassistant/components/alexa/intent.py

```diff
@@ -40,6 +40,15 @@ def async_setup(hass):
     hass.http.register_view(AlexaIntentsView)
 
 
+async def async_setup_intents(hass):
+    """
+    Setup intents.
```

Comment:
I have no glue what that means:

```
homeassistant/components/alexa/intent.py:44:1: D401 First line should be in imperative mood; try rephrasing
```

Reply:
```suggestion
    Do intents setup.
```

https://stackoverflow.com/questions/59902102/why-is-imperative-mood-important-for-docstrings

i think it s only a documentation issue ( only googled it )